.gitattributes

```diff
@@ -0,0 +1,63 @@
+# Code
+*.py text eol=lf diff=python
+*.html text diff=html
+*.js text
+*.css text diff=css
+*.sh text eol=lf diff=bash
+*.mako text
+*.mak text linguist-language=Mako
+
+
+# Assets
+*.png binary
+
+# Configuration
+*.cfg text
+*.conf.sample
+*.yaml text
+*.yml text
+*.json text
+*.json.sample text
+*.toml text
+*.ini text
+*.ini.sample text
+*.in text
+requirements.txt text
+
+# Documentation
+LICENSE text
+CHANGELOG text
+*.md text diff=markdown
+
+# To ignore from release
```

Comment:
Just an exemple of usage of export-ignore, not sure we should keep this.

Reply:
I don't think so, especially editor configurations, which is good to share across the team